apps/nextjs-website/src/components/PressReleaseList.tsx

```diff
@@ -6,6 +6,23 @@ import { LocalizeURL } from '@/lib/linkLocalization';
 import { PressReleaseList as PressReleaseListRC } from '@react-components/components';
 import { PressReleaseListProps } from '@react-components/types';
 
+const formatDateToLocale = (
+  dateString: string,
+  locale: string = 'en-GB'
```

Comment:
```suggestion
  locale: string = 'it'
```
Let's stick to the locales we use ('it' and 'en')

Also I would default to 'it' because everything in the site generally does